.github/workflows/create-crd-update-pr-in-longhorn-repo.yml

```
@@ -39,7 +39,7 @@ jobs:
       id: cpr
       uses: peter-evans/create-pull-request@v7
       with:
-        token: ${{ secrets.CUSTOM_GITHUB_TOKEN }}
+        token: ${{ secrets.GITHUB_TOKEN }}
         branch: "update-image-tags-${{ inputs.branch }}"
```

actionlint (1.7.4)

property "branch" is not defined in object type {}

(expression)